pages/tool_assembly/NO_TSD_assembly.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Norwegian ELIXIR tools assembly for sensitive data - TSD
+contributors: [Tina Visnovska, Federico Bianchini, Korbinian Bösl]
+summary: This is a Data Management tools assembly for sensitive data around TSD. TSD as an infrastructure is aimed for researchers in Norway and their collaborators, but can be used by anyone.
+tags: [plan, collect, analyse, preserve, share, reuse, IT support, data manager, researcher, human data, DMP, storage, sensitive]
+---
+
+## What is the Norwegian tools assembly for sensitive data - TSD data management tools assembly?
+The Norwegian ELIXIR tools assembly for sensitive data is centred around 
+[TSD - literally for: services for sensitive data](https://www.uio.no/english/services/it/research/sensitive-data/ is an infrastructure provided by [the University of Oslo (UiO)](https://www.uio.no). Together with the other complementary tools provided by ELIXIR, TSD can be used for the management of [sensitive data](sensitive_data), including handling of [Human data](human_data).
+This assembly covers [Planning](planning), [Processing](processing), [Analysing](analysing) and [Sharing](sharing) Data Life Cycle stages and offer [Data Storage](storage) capacities and tools for [transfer](data_transfer) of sensitive data, following the requirements of the EU general data protection regulations (GDPR) and its Norwegian implementation. 
+
+
+
+## Who can use the TSD data management tools assembly?
+
+Resources on TSD itself are accessible for international users through [EOSC](https://marketplace.eosc-portal.eu/services/tds) and directly through the [University of Oslo](https://www.uio.no/english/services/it/research/sensitive-data/access/). Researchers in Norway can in addition obtain access through the [national e-infrastructure program Sigma2](https://www.sigma2.no/sensitive-data-services) and additional storage quotas (up to 10TB) for existing TSD projects through the (Norwegian) bioinformatics support desk [contact@bioinfo.no](mailto:contact@bioinfo.no).
+If you are affiliated to a Norwegian institution which has already stipulated a [data processing and/or service agreement with TSD](https://www.uio.no/tjenester/it/forskning/sensitiv/hjelp/start/kontrakter/index.html), this document can be used or amended to account for your project. Otherwise, you can establish a data agreement for your individual project by contacting [tsd-contact@usit.uio.no}(mailto:tsd-contact@usit.uio.no).
+
+## What can you use the TSD data management tools assembly for?
```

Comment:
CSC - Finland tool assembly gave subtitles to the paragraphs in this section, according to the RDM stage or problem. It might be useful.